lib/parzival_web/live/backoffice/tools/scanner_live/index.html.heex

```diff
@@ -3,26 +3,14 @@
     Scan a QR code
   </h2>
 
-  <div id="loadingMessageWrapper" class="pt-5 pr-6 pb-5 pl-4 mt-16 sm:pl-6 lg:pl-8 xl:pt-6 xl:pl-6">
-    <div class="flex justify-between items-center h-10">
+  <div class="my-8 w-full lg:px-6 lg:mx-16">
+    <div id="qr-scanner" phx-hook="QrScanner" data-ask_perm="perm_button" data-open_on_mount data-on_start="document.getElementById('request').style.display = 'none'" data-on_success="window.location = 'http://localhost:4000/app/profile/' + uuid" class="relative"></div>
```

Comment:
```suggestion
    <div id="qr-scanner" phx-hook="QrScanner" data-ask_perm="perm_button" data-open_on_mount data-on_start="document.getElementById('request').style.display = 'none'" data-on_success="window.location = 'http://join.di.uminho.pt/app/profile/' + uuid" class="relative"></div>
```

Reply:
Nice catch. I think I can make it hostname-agnostic